src/Tests/Microsoft.NET.Pack.Tests/Microsoft.NET.Pack.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project>
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <NonShipping>true</NonShipping>
+    <OutDirName>Tests\$(MSBuildProjectName)</OutDirName>
```

Comment:
>Tests [](start = 16, length = 5)

I'd remove this. What's the benefit of building tests to a Tests subdirectory?

Reply:
less changes required to the existing test framework used by the SDK repo.
I have a local list of a number of things which need further cleanup